modules/yugabytedb/yugabytedb_test.go

```diff
@@ -0,0 +1,108 @@
+package yugabytedb_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/yugabyte/gocql"
+
+	_ "github.com/lib/pq"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/yugabytedb"
```

Comment:
Nit: same here about the imports and golangci-lint

Reply:
I've ran the linter but no issues on my side